lib/bundler/inject/dsl_patch.rb

```diff
@@ -97,6 +100,10 @@ def load_bundler_d(dir)
           eval_gemfile(f)
         end
       end
+
+      def search_path(gemfile_dir)
```

Comment:
I'm wondering if this should just be inlined into the expand_gem_path method, since it's not being reused anywhere else.  The reason is that this method will be patched into the Bundler::Dsl module, and the generic name of the method concerns me that it might conflict in the future.